content/en/bits_ai/bits_ai_dev_agent/setup.md

```diff
@@ -7,38 +7,30 @@ disable_toc: false
 
 Bits AI Dev Agent integrates with GitHub to open, update, and iterate on pull requests based on issues detected in Datadog.  
 
-<div class="alert alert-info">Bits AI Dev Agent supports GitHub only.</div>
-
 ## Setup
 
-The following steps are required to get started with Bits AI Dev Agent. 
-
-### Step 1: Enable the GitHub integration
-
-Install the [GitHub integration][5]. For full installation and configuration steps, see the [GitHub integration guide][6].
-
-### Step 2: Configure GitHub permissions
+1. Install the [GitHub integration][5]. For full installation and configuration steps, see the [GitHub integration guide][6].
 
-The GitHub integration must be granted the following permissions to enable basic Dev Agent functionality:
+1. Configure GitHub permissions:
```

Comment:
```suggestion
1. In Datadog's GitHub integration, configure the following GitHub permissions:
```
Thought this might clarify that you're getting two steps about adding permissions in the same place! Then someone's less likely to go from step 1 to 2 thinking that the break in steps means that they're configured somewhere else, and in both cases they (hopefully) won't be going to GitHub to configure their permissions there.